fetch-publication/src/test/java/no/unit/nva/publication/fetch/FetchPublicationHandlerTest.java

```diff
@@ -1,6 +1,32 @@
 package no.unit.nva.publication.fetch;
 
-import static java.util.Collections.singletonMap;
+import com.amazonaws.services.lambda.runtime.Context;
```

Comment:
Line 58: output should be of `ByteArrayOutputStream` since this is the method that overides the default toString() method and we are making use of. Any other OutputStream will fail in the tests